build_tools/packaging/linux/README.md

```diff
@@ -0,0 +1,28 @@
+Prerequisites:
+
+Python version required:  python 3.12.3
+Install rpm package
+Ex: apt install rpm (in Ubuntu)
+pip install -r requirements.txt
+
+usage: build_package.py [-h/\s\+$//e --artifact-url ARTIFACT_URL --dest-dir DEST_DIR [--gfx-arch GFX_ARCH] [--pkg-type PKG_TYPE] [--rocm-version ROCM_VERSION]
+                        [--version-suffix VERSION_SUFFIX] [--install-prefix INSTALL_PREFIX] [--rpath-pkg] [--clean-build]
+                        [--pkg-names PKG_NAMES [PKG_NAMES ...]]
+
+options:
```

Comment:
Is this useful to inline into the README? Running `build_package.py --help` should be accurate and less of a maintenance burden (otherwise if we add a new option this document will be outdated)